docs/swagger/spec.yml

```diff
@@ -0,0 +1,331 @@
+        - name: organization
+          in: body
+          required: true
+          schema:
+            $ref: '#/definitions/Organization'
+      responses:
+        201:
+          description: Return newly created organization
+          schema:
+            $ref: '#/definitions/Organization'
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+  /organizations/{organizationSlug}/:
+    get:
+      summary: Retrieve details for an organization
+      tags:
+        - Organizations
+      parameters:
+        - name: organizationSlug
+          in: path
+          required: true
+          type: string
+      responses:
+        200:
+          description: Returned organization
+          schema:
+            $ref: '#/definitions/Organization'
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+    put:
+      summary: Update an organization
+      tags:
+        - Organizations
+      parameters:
+        - name: organization
+          in: body
+          required: true
+          schema:
+            $ref: '#/definitions/Organization'
+        - name: organizationSlug
+          in: path
+          required: true
+          type: string
+      responses:
+        204:
+          description: Update successful (No Content)
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+
+parameters:
+  orderingBase:
+    name: ordering
+    in: query
+    description: Field to order results by; meaning of label varies based on model
+    type: string
+    enum:
+      - createdAt
+      - modifiedAt
+  organization:
+    name: organization
+    in: query
+    description: Slug for organization to filter by
+    type: string
+  page:
+    name: page
+    in: query
+    description: Page of results to go to
+    type: number
+    format: int32
+  pageSize:
+    name: pageSize
+    in: query
+    description: Number of results per page in paginated response
+    type: number
+    format: int32
+  uuid:
+    name: uuid
+    in: path
+    required: true
+    type: string
+    format: uuid
+
+definitions:
+  BaseModel:
+    type: object
+    readOnly: true
+    properties:
+      uuid:
+        type: string
+        description: unique identifier for object
+        format: uuid
+      organization:
+        type: string
+        description: slug label for organization
+      createdAt:
+        type: string
+        description: timestamp of object creation
+        format: date-time
+      modifiedAt:
+        type: string
+        description: timestamp of object modificiation
+        format: date-time
+  User:
+    allOf:
+      - $ref: '#/definitions/BaseModel'
+    type: object
+    required:
+      - userID
+      - role
+    properties:
+      userID:
+        type: string
+        description: Unique identifier provided by Auth0
+      is_active:
```

Comment:
Should this be 'isActive', since js/scala both prefer camel-case over underscores?